src/features/SolarSystem/SolarRender.h

```diff
@@ -0,0 +1,345 @@
+#pragma once
+
+#include <Renderer.h>
+#include <gl\GL.h>
+#include <gl\GLU.h>
+#include"features\SolarSystem\SolarResource.h"
+namespace Features
+{
+	using namespace Interfaces;
+
+	class SolarSystemRenderer : public IRenderer
+	{
+	private:
+		float		g_Angle;
+		float 		g_Mercury;
+		float 		g_Venus;
+		float 		g_Earth;
+		float 		g_Mars;
+		float 		g_Jupiter;
+		float 		g_Saturn;
+		float 		g_Uranus;
+		float 		g_Neptune;
+		GLuint		gbTexture[10];
+		GLUquadric	*pQuadric[9];
+	public:
+		void update()
+		{
+			g_Angle = g_Angle + 0.1f;
+			if (g_Angle >= 360)
+				g_Angle = 0.1f;
+
+			g_Mercury = g_Mercury + 0.7f;
+			g_Venus = g_Venus + 0.3f;
+			g_Earth = g_Earth + 0.5f;
+			g_Mars = g_Mars + 0.3f;
+			g_Jupiter = g_Jupiter + 0.5f;
+			g_Saturn = g_Saturn + 0.6f;
+			g_Uranus = g_Uranus + 0.7f;
+			g_Neptune = g_Neptune + 0.5f;
+		}
+		int LoadTexture(GLuint *texture, TCHAR resource[])
```

Comment:
Wouldn't be better to use SOIL for loading the image file?
If you use SOIL, then you can load more compressed image types which can save disk-IO and get us faster loading time.